recipes/hazelcast-cpp-client/all/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.10)
+project(cmake_wrapper)
+
+if (MSVC)
+    add_compile_definitions("/EHsc")
+endif()
```

This should be all it takes (I can not see if you tried this) if it does not work we can look into why

```suggestion
include(conanbuildinfo.cmake)
conan_basic_setup()
```